src/utils.ts

```diff
@@ -23,3 +24,60 @@ export function normalizePath(currentPath: string, basePath: string): string {
   const result = path.join(basePath, currentPath);
   return result;
 }
+
+export function uniquePathWithMember(ownerName: string, memberName: string): string {
+  return `${capitalize(ownerName)}Members${capitalize(memberName)}Type`;
+}
+
+export function uniquePathWithMethodParameter(ownerName: string, methodName: string, parameterName: string): string {
+  return `${capitalize(ownerName)}${capitalize(methodName)}${capitalize(parameterName)}`;
+}
+
+export function uniquePathWithMethodReturnType(ownerName: string, methodName: string): string {
+  return `${capitalize(ownerName)}${capitalize(methodName)}ReturnType`;
+}
+
+export function basicTypeOfUnion(union: UnionType): BasicTypeValue {
+  const { type } = union.members[0];
+  if ('value' in type) {
+    return type.value;
+  }
+  return BasicTypeValue.string;
+}
+
+export function membersOfUnion(union: UnionType): EnumField[] {
+  const result: EnumField[] = [];
+  union.members.forEach((value) => {
```

Comment:
Based on the definition of `LiteralType`, boolean type and enum type are not handled here. This would cause the generated code missing some cases, or even a compiling error if none of the members is handled.

Reply:
They are removed